### ISSUE TITLE
[Bug] aws supporting services is not working

### ISSUE THREAD

**Issue by jonormann** [dynatrace-oss/terraform-provider-dynatrace#73]
Hi,

I tried to add kafka as supporting_services_to_monitor in an aws config.
to do that I need to add at least all 50 monitored_metrics blocks that are recommended. Same when I do that via API.
Via API works but with terraform I get this error:
```
Error: Too many monitored_metrics blocks
│ 
│   on modules/environment/aws_credentials.tf line 63, in resource "dynatrace_aws_credentials" "aws":
│   63:     monitored_metrics {
│ 
│ No more than 10 "monitored_metrics" blocks are allowed
```
If I create the supporting service manually and download the tf via the provider, also all monitored_metrics blocks are inclueded.
I think the constraint to only 10 blocks is not correct here.

**Comment by Dynatrace-Reinhard-Pilz** [comment]
You're right. I just had a look at it, turns out that it was a copy&paste error. This will be fixed with v1.8.0.

**Comment by jonormann** [comment]
You are realy fast in fixing things, that is perfect!
